test/types/lib.spec.ts

```diff
@@ -49,28 +53,28 @@ const main = async () => {
     ],
     custom_claim: "CustomValue"
   });
-  expectNotAssignable<OktaJwtVerifier.JwtClaims>({
+  expect<OktaJwtVerifier.JwtClaims>().type.not.toBeAssignable({
     exp: 'not-a-number'
   });
+});
 
-  // JWT Header
-  expectAssignable<OktaJwtVerifier.JwtHeader>({
-    alg: 'RS256',
+test("JWT Header", () => {
+  expect<OktaJwtVerifier.JwtHeader>().type.toBeAssignable({
+    alg: 'RS256' as const,
```

Comment:
`as const` is needed here, because TSTyche assertions always widen types.

Behaviour of `tsd`s assertions is the following: `expectAssignable` narrows types based on the expected type, but `expectNotAssignable` always widens types. In other words, both following assertions are passing (does not look right to me):

```ts
expectAssignable<OktaJwtVerifier.JwtHeader>({ alg: 'RS256', typ: 'JWT' });
expectNotAssignable<OktaJwtVerifier.JwtHeader>({ alg: 'RS256', typ: 'JWT' });
```